vision/product_search/README.md

```diff
@@ -0,0 +1,37 @@
+# Google Cloud Vision API Product Search Go example
+
+## Authentication
+
+* Create a project with the [Google Cloud Console][cloud-console], and enable
+  the [Vision API][vision-api].
+* From the Cloud Console, create a service account,
+  download its json credentials file, then set the 
+  `GOOGLE_APPLICATION_CREDENTIALS` environment variable:
+
+  ```bash
+  export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
+  ```
+
+[cloud-console]: https://console.cloud.google.com
+[vision-api]: https://console.cloud.google.com/apis/api/vision.googleapis.com/overview?project=_
+[adc]: https://cloud.google.com/docs/authentication#developer_workflow
+
+## Run the samples
```

Comment:
The samples do not need to be able to run. The functions should either stand alone or be a quickstart.

Reply:
done